Description: Use Debian's copyright file
 Don't reference a nonexistent upstream LICENSE file.
Author: Benjamin Barenblat <bbaren@mit.edu>
Forwarded: not-needed
--- a/README
+++ b/README
@@ -25,7 +25,7 @@ COPYRIGHT:
 The CompCert verified compiler is Copyright 2004, 2005, 2006, 2007,
 2008, 2009, 2010, 2011, 2012, 2013, 2014 Institut National de Recherche en
 Informatique et en Automatique (INRIA).  It is distributed under the
-conditions stated in file LICENSE.
+conditions stated in file /usr/share/doc/compcert/copyright.
 
 
 CONTACT:
